integration-test/2022-disputed-borders.py

```diff
@@ -96,16 +99,14 @@ def test_admin_level_3_country(self):
                 'boundary': 'administrative',
                 'place': 'country',
                 'source': 'openstreetmap.org',
-                'admin_level:YY': '2'
+                'admin_level:XX': '2',
+                'recognized_by': 'YY',
             }),
         )
 
-        self.assert_has_feature(
```

Comment:
Ditto on stale admin_level = 3 test.

Reply:
Fixed in 9506b97782500824588c5745c04dc520b202da17